fpdf/fpdf.py

```diff
@@ -1050,7 +1048,8 @@ def multi_cell(
             link (str): optional link to add
             ln (int): Indicates where the current position should go after the call.
                 Possible values are: `0`: to the right ; `1`: to the beginning of the next line ;
-                `2`: below. Putting 1 is equivalent to putting 0 and calling `ln` just after.
+                `2`: below ; `3`: to the right .
```

Comment:
Is this just a copy/paste mistake or are `0` and the new value `3` behaving the same way and both are "to the right"? 🤔

Reply:
It was incomplete :) This is now better detailed.